src/react/workflow/ReplicationForm.tsx

```diff
@@ -54,6 +79,22 @@ function ReplicationComponent({
 
   const { register, errors, control, reset, getValues } = useFormContext();
 
+  const state = useSelector((state: AppState) => state);
```

Comment:
I think the import of AppState is missing in this class

Reply:
good catch I just added it